helm/charts/determined/values.yaml

```diff
@@ -196,7 +196,8 @@ checkpointStorage:
   # system.
   type: shared_fs
   hostPath: /checkpoints
-
+  # By default don't mount the shared_fs to the master pod. change this to true to enabled checkpoint downloads from master
+  mountToMaster: false
```

Comment:
# By default, shared_fs is not mounted to the master pod. Change this to true to enable checkpoint downloads from the master.